code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5549,3 +5549,108 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	reagent_state = SOLID
 	color = "#FFA500"
 	custom_metabolism = 0.1
+
+//Plant-specific reagents
+
+/datum/reagent/kelotane/tannic_acid
+	name = "Tannic Acid"
+	id = TANNIC_ACID
+	description = "Tannic acid is a natural burn remedy."
+	reagent_state = LIQUID
+	color = "#C8A5DC" //rgb: 200, 165, 220
+
+/datum/reagent/dermaline/kathalai
+	name = "Kathalai"
+	id = KATHALAI
+	description = "Kathalai is an exceptional natural burn remedy, it performs twice as well as tannic acid."
+	color = "#C8A5DC" //rgb: 200, 165, 220
+
+/datum/reagent/bicaridine/opium
+	name = "Opium"
+	id = OPIUM
+	description = "Opium is an exceptional natural pain reliever."
+	color = "#C8A5DC" //rgb: 200, 165, 220
+
+/datum/reagent/space_drugs/mescaline
+	name = "Mescaline"
+	id = MESCALINE
+	description = "Known to cause mild hallucinations, mescaline is often used recreationally."
+	color = "#60A584" //rgb: 96, 165, 132
+
+/datum/reagent/synaptizine/nicotine
+	name = "Nicotine"
+	id = NICOTINE
+	description = "Nicotine is used to improve cognition, alertness, and focus."
```

Comment:
Wikipedia cited [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3710300/) as the source for that bit of information.

I mean, obviously no doctor is going to tell you to use it, since it's highly addictive, but it *does* have those effects.